index.html

```diff
@@ -2,21 +2,25 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
     <script src="https://kit.fontawesome.com/d52fa491d2.js" crossorigin="anonymous"></script>
+    <title>Document</title>
 </head> 
 <body> 
     <header>
-        <nav> 
-            <a href="#">My Logo</a>
-            <i class="fa-solid fa-bars"></i>
-        </nav>
+        <ul>
+            <li><a href="#">My Logo</a></li>
+            <li><i class="fa-solid fa-bars"></i></li>   
```

Comment:
- [ ] Your hamburger button is not positioned correctly, please try to fix that and I suggest aligning the content of the header with `Flex box` in order to make sure everything stays in its place 
![image](https://user-images.githubusercontent.com/61790587/177768779-f3f2c0b9-3d98-4000-b72f-805fb8ef5e71.png)

Reply:
> * [ ]  As required by the previous reviewer, Your hamburger button is not positioned correctly, please try to fix that and I suggest aligning the content of the header with `Flex box` in order to make sure everything stays in its place